src/utils/wallet.js

```diff
@@ -637,7 +637,7 @@ class Wallet {
         return availableKeys;
     }
 
-    async getAccountBasic(accountId) {
+    getAccountBasic(accountId = this.accountId) {
```

Comment:
I don't think that we want to add any implied logic here; it makes it too easy for someone to call `getAccountBasic(variableName)` where `variableName` happens to be a reference to an undefined key on an object that they expected to be populated... which then becomes a silent fallback for this function to use the account that happens to be set on the `wallet` object instance (which may or may not make any sense in the context of the caller of `getAccountBasic()`).  I'd rather that bug be discovered when `account.state()` throws an error than when we use the _result_ of `account.state()` for an action that turns out to be using _the wrong account_ for the action...

Reply:
Got it, done.